lib/Runtime/Base/ScriptContext.cpp

```diff
@@ -2743,11 +2743,11 @@ if (!sourceList)
 
         hr = this->GetDebugContext()->RundownSourcesAndReparse(shouldPerformSourceRundown, /*shouldReparseFunctions*/ true);
 
-        // Debugger attach/detach failure is catastrophic, take down the process
-        DEBUGGER_ATTACHDETACH_FATAL_ERROR_IF_FAILED(hr);
-
         if (!this->IsClosed())
```

Comment:
I am okay to move the FATAL error inside, but why we are proceeding when the this is closed? why not bail out from this function in that case?